spec/PhpSpec/Matcher/ThrowMatcherSpec.php

```diff
@@ -44,6 +44,17 @@ function it_accepts_a_method_during_which_an_error_should_be_thrown(ArrayObject
         $this->positiveMatch('throw', $arr, array('\Error'))->during('ksort', array());
     }
 
+    function it_accepts_a_method_during_which_an_error_specified_by_instance_should_be_thrown(ArrayObject $arr)
+    {
+        if (!class_exists('\Error')) {
+            throw new SkippingException('The class Error, introduced in PHP 7, does not exist');
+        }
+
+        $arr->ksort()->willThrow(new \Error());
```

Comment:
See if this works...

`$arr->ksort()->will(function(){ throw new \Error(); });`

This might keep us with a passing build until the corresponding Prophecy changes can be released